### PR TITLE
Do not fallback to dbs locations for global

### PR DESCRIPTION
**Pull request by mmascher**

### CODE REVIEW

**Comment by mmascher** [comment]
Requires new WMCore tag

**Comment by AndresTanasijczuk** [comment]
Don't you want to do `phedexOnly = self.dbsInstance.split('/')[1] == 'global'` ?

**Comment by belforte** [comment]
this releates to https://github.com/dmwm/CRABServer/issues/5013

**Comment by belforte** [comment]
@AndresTanasijczuk yes, we want to do that. But it seems equivalent to
dbsOnly = self.dbsInstance.split('/')[1] != 'global' 
at least the way DBS3Reader is coded now in WMCore.
Marco and myself were doing this last night in order to teach Stefano how to build CRAB TW rpms, so we did not think too much about the code detail, we could not merge your WMCore PR since that applies to WMCore master branch which is too way ahead of what we use in Crab (has the so called PNN megapatch in).
Basically this is "how to make a quick&dirty patch still in a fully-documented-reproducible way using git and rpm rather then edit files on TW machine".

Quit possibly the rpm's we get only land on my private VM for testing.

**Comment by AndresTanasijczuk** [comment]
Ok, I missed the commit from Marco https://github.com/mmascher/WMCore/commit/5f98eb22f7f1c8c6f61da8b97a19860d43205c4c in his WMCore.

**Comment by belforte** [comment]
amazing that you found it. As I said this was more a "let's make an example" then a real thing.
I discussed with Eric at lunch time how to deal in a more solid way with WMCore.
We can define this in next meeting.

**Comment by mmascher** [comment]
Once this https://github.com/dmwm/WMCore/pull/6286 get merged and we will get back to a clean WMCore we should add the phedexOnly flag as well as you suggested andres. Let's not merge this or the moment.

**Comment by AndresTanasijczuk** [comment]
dmwm/WMCore#6286 just got merged

**Comment by mmascher** [comment]
Thanks Andres, I suspect that validating WMCore will take more time than I have at the moment (besides I am not sure the pycurl_manager and PNN changes are finished yet..). So I will go again with my modified version of WMCore for the November release.

I'll cherrypick this commit.

**Comment by mmascher** [comment]
Replaced by https://github.com/dmwm/CRABServer/pull/5122
